src/server/tsconfig.library.json

```diff
@@ -1,29 +1,33 @@
 {
     "compilerOptions": {
         "noImplicitAny": true,
-        "removeComments": true,
+        "noImplicitThis": true,
+        "removeComments": false,
         "preserveConstEnums": true,
+        "pretty": true,
         "outFile": "../../built/local/tsserverlibrary.js",
         "sourceMap": true,
-        "stripInternal": true,
-        "declaration": true,
-        "types": [],
+        "types": [
+            "node"

```

Comment:
do not think you need `node` here. the library should not have any dependency on the environment, if it does, this is a bug that needs to be fixed.